<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>399</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT Bold</family>
    <pointsize>10</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Current Game - NoGo</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="restartButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resignButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Resign</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="TimeBar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar
{
background:rgb(186,215,233);
color:white;
border-radius:5px;
}
QProgressBar::chunk
{
border-radius:5px;
background-color:rgb(255,192,203);
}</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="chatHistory">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="chatInput">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="autoLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
